docs/moderators/github.md

```diff
@@ -0,0 +1,55 @@
+# Maintaining Repositories
+
+This page is intended to provide guidelines for maintaining EddieHub's various repositories.
+
+## When To Merge PRs
+
+Pull requests should be merged only after:
+
+- At least 2 maintainers have provided an approving review.
```

Comment:
I guess @nhcarrigan mentioned about 1 approval to merge above. So should we have 1 here?

Reply:
None other than maintainers can merge, so a merge is considered a approval 🤔

Reply:
I think we can also have a provision of -> Many approvals from the community + one approval from a maintainer == PR can be merged

Reply:
> I think we can also have a provision of -> Many approvals from the community + one approval from a maintainer == PR can be merged

This sounds good